peak2bw.py

```diff
@@ -52,14 +56,24 @@ def parse_args():
 
     """
     parser = argparse.ArgumentParser(
-        description='Convert BED file or MACS narrowPeak file to bigWig file')
-    parser.add_argument('input', type=str, help='Path to narrowPeak file')
-    parser.add_argument('sizes', type=str,
-                        help='Path to chromosome sizes file')
-    parser.add_argument('--prefix', type=str, help='Output file prefix')
+        description='Convert BED file or MACS2 narrowPeak file \
+        to bigWig file')
+    parser.add_argument('input', type=str, required=True,
+                        help='Path to narrowPeak or BED file')
+    parser.add_argument('sizes', type=str, required=True,
+                        help='Path to chromosome sizes file. \
+                        Only peaks in these chromosomes will \
+                        be encoded in the output bigWig file.')
+    parser.add_argument('--prefix', type=str, help='Output file \
```

Comment:
Do we have a out_home ? Otherwise, we need to make it clear that `--prefix` needs to have `full-path/prefix` format. Or, let's add a out_home argument.